django/librarian/utils/markdown_splitter.py

```diff
@@ -100,9 +100,14 @@ def _close_page_tags(self, html_string: str) -> str:
         * chunk ends with <page_n+1> (with no content for that page)
         """
         html_string = html_string.strip()
+        print(f"HTML string: {html_string}")
         # Closing tag as the first line?
         if html_string.split("\n", 1)[0].startswith("</page_"):
-            html_string = html_string.split("\n", 1)[1]
+            if 1 in range(len(html_string.split("\n", 1))):
```

Comment:
The condition `if len(html_string.split("\n", 1) > 1` may be more clear.

This is the default case right? Where a closing page tag starts the chunk, but there is more content below, so the initial closing tag is simply removed?
Might help to add some comments. (I should have done so before).